pylint_odoo/checkers/modules_odoo.py

```diff
@@ -484,6 +485,16 @@ def _get_duplicate_xml_fields(self, fields):
             all_fields.setdefault(field_xml, []).append(field)
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
+            parents = [e.getparent() for e in items]
+            same_parent = [
+                item for item, count in collections.Counter(parents).items()
+                if count > 1]
+            # Remove elements which are in different parent element
+            # Parents in items but not in same_parent list
+            exclude = list(set(parents) - set(same_parent))
+            for item in items:
+                if item.getparent() in exclude:
+                    items.remove(item)
```

Comment:
@hugho-ad 
Please, review the following good comments from @pedrobaeza and fix them
https://github.com/OCA/pylint-odoo/pull/63#r75580525

We have the same case here